src/meetingsession/DefaultMeetingSession.ts

```diff
@@ -61,6 +63,8 @@ export default class DefaultMeetingSession implements MeetingSession, Destroyabl
       ),
       this._eventReporter
     );
+    this.cspSubscriber = new DefaultCSPSubscriber(this._logger);
```

Comment:
See my comments in the CSP component. You can call:

```
this.cspViolationDetector.start();
```

In the `destroy` funtion:
```
this.cspViolationDetector.stop();
```

Reply:
Yep, will do.